packages/api-reference/src/components/Content/ReferenceEndpoint/RequestBody.vue

```diff
@@ -1,21 +1,39 @@
 <script setup lang="ts">
+import { ref } from 'vue'
+
 import type { RequestBody } from '../../../types'
 import Schema from '../Schema.vue'
 
-defineProps<{ requestBody?: RequestBody }>()
+const prop = defineProps<{ requestBody?: RequestBody }>()
+const selectedContentType = ref(
+  prop?.requestBody?.content
+    ? Object.keys(prop?.requestBody?.content)[0]
+    : undefined,
+)
 </script>
 <template>
-  <div v-if="requestBody && requestBody.content?.['application/json']">
+  <div v-if="prop?.requestBody">
     <div class="request-body-title">
       <slot name="title" />
+      <select
```

Comment:
@cameronrohani would you be able to come in and bring in the styles we use for `selects` or guide on where we should be pulling from :)